lib/node_modules/@stdlib/math/base/assert/is-probability/README.md

```diff
@@ -74,6 +74,97 @@ for ( i = 0; i < 100; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/math/base/assert/is_probability.h"
+```
+
+#### stdlib_base_is_probability( x )
+
+Tests if a `numeric` value is a probability.
+
+```c
+bool out = stdlib_base_is_probability( 0.5 );
+// returns true
+
+out = stdlib_base_is_probability( 3.14 );
+// returns false
+```
+
+The function accepts the following arguments:
+
+-   **x**: `[in] double` input value.
+
+```c
+bool stdlib_base_is_probability( const double x );
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- C API usage examples. -->
+
+<section class="examples">
+
+### Examples
+
+```c
+#include "stdlib/math/base/assert/is_probability.h"
+#include <math.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+int main() {
+    double x;
+    bool v;
```

Comment:
Don't we need `stdbool.h` for `bool` macro support?

Reply:
Yeah! added it.